src/unix/special.c

```diff
@@ -118,8 +118,14 @@ void register_special_files(process p)
         assert(proc_self);
         value program = table_find(p->process_root, sym(program));
         assert(program);
-        buffer b = clone_buffer(h, program);
+        buffer b;
```

Comment:
Just a nit, but I would have just changed clone to allocate_buffer of size buffer_length((buffer)program) + 2 (to avoid buffer realloc), then just conditionally buffer_write_byte(b, '/') if program doesn't start with a slash. But really what's there is fine...

Reply:
Yeah, that's a lot nicer way to do it, thanks!